File: tests/py-to-cpp/pub.py
from message import zmq_clang_pub as zmq_pub
import time

while True:
    init = zmq_pub.publish("logs_test_py_to_py")
    init.new_message(size=1)
    init.send(["Temp"], ["67°F"])
    print("Sent")
    time.sleep(1)

File: tests/cpp-to-py/pub.cc
#include<message/zmq_clang_pub.h>
#include<unistd.h>
int main()
{
    while(true)
    {
        publish_to_py publisher;
        Json::Value rootl;
        rootl["temp"] = "67°F";
        publisher.new_message("Logs_cxx_to_py");
        publisher.send_json(rootl);
        std::cout<<"Sent message"<<std::endl;
        sleep(1);
    }
    return 0;
}


